packages/hash/api/src/graphql/resolvers/ontology/entity-type.ts

```diff
@@ -0,0 +1,122 @@
+import { ApolloError } from "apollo-server-express";
+import { AxiosError } from "axios";
+
+import {
+  PersistedEntityType,
+  MutationCreateEntityTypeArgs,
+  MutationUpdateEntityTypeArgs,
+  QueryGetEntityTypeArgs,
+  Resolver,
+} from "../../apiTypes.gen";
+import { GraphQLContext } from "../../context";
+import { EntityTypeModel } from "../../../model";
+import { nilUuid } from "../../../model/util";
+import { entityTypeModelToGQL } from "./model-mapping";
+
+export const createEntityType: Resolver<
+  Promise<PersistedEntityType>,
+  {},
+  GraphQLContext,
+  MutationCreateEntityTypeArgs
+> = async (_, params, { dataSources }) => {
+  const { graphApi } = dataSources;
+  const { accountId, entityType } = params;
+
+  const createdEntityTypeModel = await EntityTypeModel.create(graphApi, {
+    accountId,
+    schema: entityType,
+  }).catch((err: AxiosError) => {
+    if (err.response?.status === 409) {
+      throw new ApolloError(
+        `Entity type with the same URI already exists. [URI=${entityType.$id}]`,
+        "CREATION_ERROR",
+      );
+    }
+
+    throw new ApolloError(`Couldn't create entity type.`, "CREATION_ERROR");
```

Comment:
Nit:
```suggestion
    const msg =
      err.response?.status === 409
        ? `Entity type with the same URI already exists. [URI=${entityType.$id}]`
        : `Couldn't create entity type.`;

    throw new ApolloError(msg, "CREATION_ERROR");
```

Reply:
Addressed in bda8c35b8711a8eda443830b74f6cfb6c75c7003 for all ontology types